redirects/pages.yml

```diff
@@ -658,4 +658,7 @@
   to: https://www.jetbrains.com/idea/whatsnew/2021-1/#kotlin-x
 
 - from: /docs/whatsnew-plugin-20203.html
-  to: https://www.jetbrains.com/idea/whatsnew/2020-3/#kotlin-x
\ No newline at end of file
+  to: https://www.jetbrains.com/idea/whatsnew/2020-3/#kotlin-x
+
+- from: /docs/mpp-intro.html
```

Comment:
This will not work. Cause, you have written `accepts-web-file-names`.
When pages exist in redirects and **WH**, pages from **WH** has priority.